thold_functions.php

```diff
@@ -2348,23 +2353,46 @@ function thold_check_threshold(&$thold_data) {
 			db_execute_prepared('UPDATE thold_data
 				SET thold_alert = ?,
 				thold_fail_count = ?,
-				thold_warning_fail_count = 0
+				thold_warning_fail_count = ?
 				WHERE id = ?',
-				array($thold_data['thold_alert'], $thold_data['thold_fail_count'], $thold_data['id']));
+				array($thold_data['thold_alert'], $thold_data['thold_fail_count'], $thold_data['thold_warning_fail_count'], $thold_data['id']));
+
+		/* Warning status */
 		} elseif ($warning_breach_up || $warning_breach_down) {
 			$notify = false;
 
-			thold_debug('Threshold HI / Low Warning check breached HI:' . $thold_data['thold_warning_hi'] . '  LOW:' . $thold_data['thold_warning_low'] . ' VALUE:' . $thold_data['lastread']);
+			thold_debug('Threshold HI / Low Warning check breached HI:' . $thold_data['thold_warning_hi'] . '  LOW:' . $thold_data['thold_warning_low'] . ' VALUE:' . $thold_data['lastread']
+				 . ' TRIGGER:' . $trigger . ' warn count:' . $thold_data['thold_warning_fail_count']);
```

Comment:
"warn count:" should be something like "WRNCNT:".  No spaces, almost like JSON.  Makes it easier for tools like Splunk to parse.

Reply:
This is changed.  Let me know what needs to be done.